packages/react-components/react-table/src/components/TableResizeHandle/useTableResizeHandleStyles.styles.ts

```diff
@@ -26,6 +26,12 @@ const useStyles = makeStyles({
     transitionDuration: '.2s',
     zIndex: 1,
 
+    ':focus': {
```

Comment:
Personally this looks kinda ugly to me 😅should we reach out to designers and ask for some guidance there?

![image](https://github.com/microsoft/fluentui/assets/20744592/bf332fce-72f1-4d33-8e36-3c14599f94eb)

Reply:
looks like a cigarette, not a good look for sure.

I'll see if I can get rid of the focus on the header itself in this case :))

Reply:
I got rid of the colliding style, but I'm still not 100% happy with the result. When entering the mode with a keyboard, the header is now focused instead of the handle, but there is no indication that you can resize. Focusing the handle would be best, but since its a child of header, which has `data-fui-focus-within` property, the header is always focused too.

I'll look into this some more.